examples/step-68/step-68.cc

```diff
@@ -334,47 +333,45 @@ namespace Step68
     const typename parallel::distributed::Triangulation<dim>::CellStatus status)
     const
   {
-    // We do not assign any weight to cells we do not own (i.e., artificial
-    // or ghost cells)
-    if (!cell->is_locally_owned())
-      return 0;
+    // First, we introduce a base weight that will be assigned to every cell.
+    constexpr unsigned int base_weight = 1;
 
     // This determines how important particle work is compared to cell
```

Comment:
While you're there, could you replace "This" by "The following variable then"?

Reply:
Also, I introduced `constexpr` here which really is not necessary. I'll change it back to const.